app/user/[userID]/profile/page.tsx

```diff
@@ -0,0 +1,95 @@
+'use client';
+
+import { useEffect, useState } from 'react';
+import Link from 'next/link';
+import supabase from '../../../../lib/supabase';
+
+export default async function UserProfile() {
+  interface User {
+   age: number,
+   city: string,
+   created_at: string,
+   ec_name: string,
+   ec_phone: string,
+   email: string,
+   first_name: string,
+   last_name: string,
+   password: string,
+   phone: string,
+   photo: string,
+   state: string,
+   street: string,
+   user_id: number,
+   zip: string
+}
+
+  const [user, setUser] = useState<Array<User> | null>([]);
+  // const [user, setUser] = useState<User | null>(null);
+  async function getUserInfo() {
```

Comment:
Type Error because function is not returning anything, can use linter ignore next line